packages/frontend/pages/api/isValidAddress.ts

```diff
@@ -18,8 +20,16 @@ const handleRequest = async (req: NextApiRequest, res: NextApiResponse) => {
     )
 
     res.status(200).json({ valid: (data?.[0]?.rating ?? 'highRisk') !== 'highRisk' })
```

Comment:
what will happen, if you get a 404 to to the call in line 10?

Reply:
All errors not 200 goes Into the catch block...I tested for 400, 403, 404.